### ISSUE TITLE
Feature Request: Prevent users from adding more than 20 columns

### ISSUE THREAD

**Issue by dance2die** [devhubapp/devhub#67]
# 🙏 Feature Request

Would you be able to prevent users from adding more than 20 columns by disabling the "➕" button?

Current behavior looks as shown below
- Some columns fail with following message

> ⚠ Too many columns
You have reached the limit of 20 columns. This is to maintain a healthy usage of the GitHub API.

![image](https://user-images.githubusercontent.com/8465237/50349860-2977b180-050b-11e9-978f-4cf17b7e742c.png)

# ➕ Additional Context

Not sure if it's the GitHub issue or DevHubApp throttling the API call as it's been working until this morning.. 🤔

**Comment by brunolemos** [comment]
I had to make some changes (#56) because some users were triggering the GitHub API Abuse detection. 

Yes we can disable the Add button! Added the Good First Issue label in case someone wants to pick this up before I do.

**Comment by dance2die** [comment]
Thank you~

**Comment by mrunmaipatil47** [comment]
Can I take this on?

**Comment by brunolemos** [comment]
@mrunmaipatil47 sure! Changes should probably be in the Sidebar.tsx file only. Theres a constant with the number of max columns allowed.

**Comment by mrunmaipatil47** [comment]
@brunolemos Have made the add columns button disappear after reaching the limit of 20 columns for now. 
Not the best UX practice, as the limit needs to be called out. 
Can continue working on disabling the button along with a tooltip(maybe?)

**Comment by mrunmaipatil47** [comment]
@brunolemos I agree. Will add the alert and disable the action of adding columns. 
Thanks.

**Comment by brunolemos** [comment]
@mrunmaipatil47 please make a pull request so I can merge

**Comment by brunolemos** [comment]
I didn't receive any pull request, so I made it.

This is how it will look in the next version:

![image](https://user-images.githubusercontent.com/619186/51230406-06dc8b00-1947-11e9-936c-a76da587dd09.png)
